tests/unit/module_utils/test_ec2.py

```diff
@@ -76,3 +78,19 @@ def test_ansible_dict_with_integer_to_boto3_filter_list(self):
 
         converted_filters_int = ansible_dict_to_boto3_filter_list(filters)
         self.assertEqual(converted_filters_int, filter_list_integer)
+
```

Comment:
@mandar242  No need to add the units and the test function inside plugins/module_utils/ec2.py  here. We are going to merge https://github.com/ansible-collections/amazon.aws/pull/597 first.